express/code/blocks/app-ratings/app-ratings.js

```diff
@@ -80,10 +111,22 @@ export default async function decorate(block) {
       replaceKey('app-store-ratings-apple-store', getConfig()),
     ],
   );
+
+  let customURL;
+  const content = block.querySelector(':scope > div a');
+  if (content) {
+    customURL = content.getAttribute('href');
+  }
+
   block.append(await makeRatings(
     ratingPlaceholder,
     starsPlaceholder,
     playStoreLabelPlaceholder,
     appleStoreLabelPlaceholder,
+    customURL,
```

Comment:
```suggestion
    content?.getAttribute('href')
```
since you have the null check in your makeRating already. This allows you to not use a "let" that is harder to reason about

Reply:
Is this a blocker? I am not comfortable changing logic when this feature has already been delayed.